recipes/tramway/meta.yaml

```
@@ -0,0 +1,61 @@
+
+{% set name = "tramway" %}
+{% set version = "0.5" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/DecBayComp/TRamWAy/archive/v{{ version }}.tar.gz
+  sha256: 6e26acc4280dfb6d93f1d5009009927dd1fa2e63f18a2c4e160b9a1e40534bc6
+
+build:
+  skip: True  # [py<36]
+  number: 0
+  # remove polytope as soon as the package is available in some conda channel, if it ever is
+  script: "{{ PYTHON }} -m pip install --no-deps https://pypi.io/packages/source/p/polytope/polytope-0.2.3.tar.gz . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - hdf5
+    - python
+    - rwa-python
+    - matplotlib
```

https://conda-forge.org/docs/maintainer/knowledge_base.html#matplotlib

```suggestion
    - matplotlib-base
```

---

Thank you for the feedback! I will take this modification if I ever find a solution for meeting the polytope dependency.